tests/ci_build/Dockerfile.i386

```diff
@@ -1,7 +1,6 @@
 FROM i386/debian:sid
 
 ENV DEBIAN_FRONTEND noninteractive
-SHELL ["/bin/bash", "-c"]   # Use Bash as shell
```

Comment:
Can we bring the line back without the comment?
```
SHELL ["/bin/bash", "-c"]
```
This should work.

Reply:
Done. Thank you for the suggestion. That's a weird bug.